src/core/dbus-unit.c

```diff
@@ -1858,6 +1859,14 @@ int bus_unit_queue_job(
             (type == JOB_RELOAD_OR_START && job_type_collapse(type, u) == JOB_START && u->refuse_manual_start))
                 return sd_bus_error_setf(error, BUS_ERROR_ONLY_BY_DEPENDENCY, "Operation refused, unit %s may be requested by dependency only (it is configured to refuse manual start/stop).", u->id);
 
+        if (u->manager && type == JOB_START && u->type == UNIT_SERVICE) {
+                Manager *m = u->manager;
+                Service *s = SERVICE(u);
+                if (manager_unit_deactivating_or_pending(m, SPECIAL_DBUS_BROKER_SERVICE) &&
```

Comment:
Why dbus-broker.service is checked instead of dbus.service? Even if dbus-broker.service is used, IIRC, we assume dbus.service is an alias of dbus-broker.service.

Also, in `signal_activation_request()`, we checked with `manager_unit_inactive_or_pending()`. Why not to also use that here?

Reply:
I think it's better to just use `manager_dbus_is_running`?